learn/what_is_meilisearch/comparison_to_alternatives.md

```diff
@@ -1,5 +1,12 @@
+---
+sidebarDepth: 0
+---
```

Comment:
This did not work in `config.js`? Just out of curiosity

Reply:
Of course, I can just do it there 🤦